### PR TITLE
docs: update docs

### PR DESCRIPTION
**Pull request by mhevery**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/pages/tutorial/hooks/use-document/problem/app.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { component$, useHostElement, Host } from '@builder.io/qwik';
+
+export const App = component$(
+  () => {
+    const host = useHostElement();
+    return <Host>Host element tag-name: {host.tagName}</Host>;
+  },
+  { tagName: 'my-app' }
+);
```
Comment (antoinepairet) [inline packages/docs/src/pages/tutorial/hooks/use-document/problem/app.tsx:9]: There seem to be a copy-paste issue as the solution and the problem are the same

File: packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { component$, useHostElement, Host } from '@builder.io/qwik';
+
+export const App = component$(
+  () => {
+    const host = useHostElement();
```
Comment (antoinepairet) [inline packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx:5]:
```suggestion
```

File: packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    return (
+      <Host onClick$={() => alert('Hello World!')}>Host element tag-name: {host.tagName}</Host>
```
Comment (antoinepairet) [inline packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx:7]:
```suggestion
      <Host onClick$={() => alert('Hello World!')}>Host element tag-name: FILL IN THE HOST TAG NAME</Host>
```

File: packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx (hunk 2 above)
Comment (mhevery) [reply packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx:5]: Thanks for the suggestion. I have incorporated it into the PR.

File: packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx (hunk 3 above)
Comment (mhevery) [reply packages/docs/src/pages/tutorial/hooks/use-host-element/problem/app.tsx:7]: Thanks for the suggestion. I have incorporated it into the PR.

File: packages/docs/src/pages/tutorial/hooks/use-document/problem/app.tsx (hunk 1 above)
Comment (mhevery) [reply packages/docs/src/pages/tutorial/hooks/use-document/problem/app.tsx]: Thanks for the suggestion. I have incorporated it into the PR.

**Pull request merged**